asheuh/Maintenance-Tracker-App
# #157831089 make sidebar responisive

#### What does this PR do?

Fix the sidebar, make it responsive

#### Description of Task to be completed?

1. Have sidebar response to every screen size

#### How should this be manually tested?
```
git clone https://github.com/asheuh/Maintenance-Tracker-App
```
```
cd Maintenance-Tracker-App
```

- Then after cloning, navigate to a file name request_list.html and open with your browser

- In your browser do the following
```
ctrl + shift + i 
```
check how the page is responsive on different screen width

#### Any background context you want to provide?
Useful tool:
[W3schools](https://www.w3schools.com/w3css/w3css_sidebar.asp)

#### What are the relevant pivotal tracker stories?
[#157831089](https://www.pivotaltracker.com/n/projects/2173420)
![request_list](https://user-images.githubusercontent.com/22955146/40569551-42b606e4-608b-11e8-8f73-1646878c7d8b.png)

![adminpage](https://user-images.githubusercontent.com/22955146/40570041-7fb187ae-608f-11e8-81db-6844a1ff34ba.png)
